### ISSUE TITLE
 app/mode-html.js  file not found ,http.status=404

### ISSUE THREAD

**Issue by antianchi** [sirensolutions/sentinl#596]
hello,
I install sentinl 6.3.0 into my kibina。I browse 

> http://127.0.0.1:5601/app/sentinl#/watcher

> ,and the console print **http://127.0.0.1:5601/app/mode-html.js net::ERR_ABORTED。**
a  404 Error.

**Comment by antianchi** [comment]
> Does it affect any functionality for you? Something doesn't work?

Yes.My kibina's sentinl is not perform well . It doesn't look like the description in [https://sentinl.readthedocs.io/en/latest/.](url)

**Comment by sergibondarenko** [comment]
It is difficult to understand your problem. Please answer all the following questions first.

**Kibana version:**

**Elasticsearch version:**

**Server OS version:**

**Browser version:**

**Browser OS version:**

**Original install method (e.g. release pkg link, gulp, from source, etc.):**

**Describe the bug:**

**Steps to reproduce:**
1.
2.
3.

**Expected behavior:**

**Sample of data you have in Elasticsearch**

**Full watcher document**

**Screenshots (if relevant):**

**Errors in browser console (if relevant):**

**Provide Kibana logs and/or server output (if relevant):**

**Any additional context:**

**Tips:**

1. Try to query Elasticsearch manually first. Do you receive the results you want for your query?
For example, on Linux, you can query Elasticsearch using `curl`:
```
curl -X GET "localhost:9200/_search?pretty" -H 'Content-Type: application/json' -d'
{
  "query": { 
    "match_all": {}
  }
}
'
```

**Comment by antianchi** [comment]
> It is difficult to understand your problem. Please answer all the following questions first.
> 
> **Kibana version:**
> 
> **Elasticsearch version:**
> 
> **Server OS version:**
> 
> **Browser version:**
> 
> **Browser OS version:**
> 
> **Original install method (e.g. release pkg link, gulp, from source, etc.):**
> 
> **Describe the bug:**
> 
> **Steps to reproduce:**
> 1.
> 2.
> 3.
> 
> **Expected behavior:**
> 
> **Sample of data you have in Elasticsearch**
> 
> **Full watcher document**
> 
> **Screenshots (if relevant):**
> 
> **Errors in browser console (if relevant):**
> 
> **Provide Kibana logs and/or server output (if relevant):**
> 
> **Any additional context:**
> 
> **Tips:**
> 
> 1. Try to query Elasticsearch manually first. Do you receive the results you want for your query?
>    For example, on Linux, you can query Elasticsearch using `curl`:
> 
> ```
> curl -X GET "localhost:9200/_search?pretty" -H 'Content-Type: application/json' -d'
> {
>   "query": { 
>     "match_all": {}
>   }
> }
> '
> ```



> It is difficult to understand your problem. Please answer all the following questions first.
> 
> **Kibana version:**
> 
> **Elasticsearch version:**
> 
> **Server OS version:**
> 
> **Browser version:**
> 
> **Browser OS version:**
> 
> **Original install method (e.g. release pkg link, gulp, from source, etc.):**
> 
> **Describe the bug:**
> 
> **Steps to reproduce:**
> 1.
> 2.
> 3.
> 
> **Expected behavior:**
> 
> **Sample of data you have in Elasticsearch**
> 
> **Full watcher document**
> 
> **Screenshots (if relevant):**
> 
> **Errors in browser console (if relevant):**
> 
> **Provide Kibana logs and/or server output (if relevant):**
> 
> **Any additional context:**
> 
> **Tips:**
> 
> 1. Try to query Elasticsearch manually first. Do you receive the results you want for your query?
>    For example, on Linux, you can query Elasticsearch using `curl`:
> 
> ```
> curl -X GET "localhost:9200/_search?pretty" -H 'Content-Type: application/json' -d'
> {
>   "query": { 
>     "match_all": {}
>   }
> }
> '
> ```
Sorry, I didn't describe it clearly.Now,I will answer your question and describe my operation.

1. Kibana version: **kibana-6.3.0-windows-x86_64**
2. Elasticsearch version:**elasticsearch-6.3.0**
3. Server OS version:**win10 Enterprise Edition 2016 LTS**
4. Browser version:**Chrome 69.0.3497.92**
5. Browser OS version:**win10 Enterprise Edition 2016 LTS**
6. Original install method (e.g. release pkg link, gulp, from source, etc.):
I installed in CMD. The command : >**kibana-plugin.bat install file:///"E:\soft\sentinl-v6.3.0 (1).zip"**.
This is my console:
`E:\soft\kibana-6.3.0-windows-x86_64\bin>kibana-plugin.bat install file:///"E:\soft\sentinl-v6.3.0 (1).zip"
Attempting to transfer from file:///E:\soft\sentinl-v6.3.0 (1).zip
Transferring 242842043 bytes....................
Transfer complete
Retrieving metadata from plugin archive
Extracting plugin archive
Extraction complete
Optimizing and caching browser bundles...
Plugin installation complete`

7. Describe the bug: When I visited [http://127.0.0.1:5601/app/sentinl#/watcher/40f5b780-dc11-11e8-acd6-1b6fc287382f/wizard?_g=()](url) .There's always something wrong with it. I press the F12 button, console print [.](vendors.bundle.js:53 GET http://127.0.0.1:5601/app/mode-html.js net::ERR_ABORTED 404 (Not Found))

8. Steps to reproduce:
When I visited [http://127.0.0.1:5601/app/sentinl#/watcher/40f5b780-dc11-11e8-acd6-1b6fc287382f/wizard?_g=()](url) .There's always something wrong with it. I press the F12 button, console print [.](vendors.bundle.js:53 GET http://127.0.0.1:5601/app/mode-html.js net::ERR_ABORTED 404 (Not Found)).
9. Expected behavior:
No 404 error.
10. Sample of data you have in Elasticsearch:**just browse**
11. Full watcher document:**No**
12. Screenshots (if relevant):
![error](https://user-images.githubusercontent.com/25560330/47711971-83799a80-dc71-11e8-9cb2-93c218ead2a6.jpg)

13. Provide Kibana logs and/or server output (if relevant):
14. Any additional context:NO

**Comment by sergibondarenko** [comment]
Ok, I understand that you see this error in the console. Now, let's ignore the error.
Do you have any other problems? Some functionality doesn't work for you?

**Comment by sergibondarenko** [comment]
1. You insert "bank" in "Select an index" input field. Do you have an index in Elasticsearch which is called "bank"? Do you see suggestions while you inserting index?
2. In order to aggregate on a date field, in "AGG date" popover, index "bank" must have at least one field of the [date type](https://www.elastic.co/guide/en/elasticsearch/reference/current/date.html). To check if there is a date field, look at the index mapping. For this, go to Kibana Dev Tools and execute `GET bank/_mapping`
